vcd/datasource_vcd_lb_server_pool.go

```diff
@@ -0,0 +1,122 @@
+package vcd
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func datasourceVcdLbServerPool() *schema.Resource {
+	return &schema.Resource{
+		Read: datasourceVcdLbServerPoolRead,
+		Schema: map[string]*schema.Schema{
+			"vdc": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "vCD organization in which the Service Monitor is located",
+			},
+			"org": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "vCD virtual datacenter in which the Service Monitor is located",
+			},
+			"edge_gateway": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Edge gateway name in which the Server Pool is located",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Existing Server Pool name",
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"algorithm": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"algorithm_parameters": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"monitor_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"enable_transparency": &schema.Schema{
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+
+			"member": {
+				Computed: true,
+				Type:     schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"condition": &schema.Schema{
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"ip_address": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"port": {
+							Type:     schema.TypeInt,
+							Computed: true,
```

Comment:
I put the descriptions into `resource`. User would never see the descriptions (because he is not inputting them) for datasources as these are computed fields. Or do you think we should add it for all? It's not hard to copy/paste and I'd add for the previous resource as well then.

Reply:
I see, makes sense. How are other providers doing it in these cases?

Reply:
vSphere has it all. Azure and AWS are missing it (at least for a few resources I saw).
In general - it's cheap to add and I guess it's quicker to add (copy/paste) it than discuss a lot :)
Maybe it helps while reading the code.

Reply:
Added